go/service/simplefs.go

```diff
@@ -24,13 +78,22 @@ const simpleFSTimeout = 1 * time.Minute
 type SimpleFSHandler struct {
 	*BaseHandler
 	libkb.Contextified
+
+	transporterWaiter *transporterWaiter
+}
+
+func (s *SimpleFSHandler) transporterGetter() (transporter rpc.Transporter) {
```

Comment:
I have a slight preference to put this below the constructor, even though the constructor references it, just because I feel like the constructor should always come first after the struct definition.  But I don't think that's a hard rule.